frontend/src/metabase-lib/segments.ts

```diff
@@ -1,10 +1,18 @@
 import * as ML from "cljs/metabase.lib.js";
+import type { SegmentId } from "metabase-types/api";
 
-import type { SegmentMetadata, Query } from "./types";
+import type { Query, SegmentMetadata } from "./types";
 
 export function availableSegments(
   query: Query,
   stageIndex: number,
 ): SegmentMetadata[] {
   return ML.available_segments(query, stageIndex);
 }
+
+export function segment(
```

Comment:
`segment_metadata` to match `table_or_card_metadata`?

Reply:
8c8769650092609cc5a373d976255eb6d4ad5bce